app/pages/neworg/createorg/template.njk

```diff
@@ -31,7 +31,8 @@
 
         <form method="post" action="{{ saveCatalogueAgreementPostAction }}">
           <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
-      
+          <input type="hidden" name="odsCode" value="{{ odsCode }}" />
```

Comment:
How come we got to do this?

Reply:
to send it to our post request where it is needed for the get to the api to get the org details in the correct format to send in the post to the api

Reply:
ah ok 👍